### ISSUE TITLE
Glob escape parsing bug for Windows

### ISSUE THREAD

**Issue by Heerschop** [chanzo/script-launcher#63]
# Escaped Glob patterns not working on Windows.

## Example 1
``` JSON
{
  "scripts": {
    "test": "echo Example: ./node_modules/\\*\\*/\\*.md"
  },
  "options": {
    "logLevel": 2
  }
}
```

### Output Windows
``` bash
Example: ./node_modules/typescript/lib/README.md
```

### Output Linux
``` bash
Example: ./node_modules/**/*.md
```

## Example 2
``` JSON
{
  "scripts": {
    "test": "echo Example: ./node_modules/\\*\\*/\\*.notfound"
  },
  "options": {
    "logLevel": 2,
    "glob": {
      "nonull": true
    }
  }
}
```

### Output Windows
``` bash
Example: ./node_modules//*/*//*.notfound
```

### Output Linux
``` bash
Example: ./node_modules/**/*.notfound
```

**Comment by chanzo** [comment]
This appears to be related to the default behavior of the [glob](https://www.npmjs.com/package/glob) package being used.
### Example Code
``` javascript
let glob = require("glob")

const options = {
  "nonull": true
};

const pattern = './node_modules/\\*\\*/\\*.md';
const result = glob.sync(pattern, options)

console.log('result:', result);
```
### Output Windows
``` cmd
result: [ './node_modules//*/*//*.md' ]
```

### Output Linux
``` bash
result: [ './node_modules/\\*\\*/\\*.md' ]
```

**Comment by Heerschop** [comment]
Workaround implemented in version:1.23.1 Should be fixed in the glob package.

**Comment by Heerschop** [comment]
Switched form glob to fast-glob in regards in version:1.24.0
